pkg/kubelet/cm/dra/manager.go

```diff
@@ -198,34 +184,40 @@ func (m *ManagerImpl) PrepareResources(pod *v1.Pod, container *v1.Container) (*C
 	return &ContainerInfo{Annotations: annotations}, nil
 }
 
-func (m *ManagerImpl) UnprepareResources(pod *v1.Pod) error {
-	m.Lock()
-	defer m.Unlock()
+// PrepareResources calls plugin NodePrepareResource from the registered DRA resource plugins.
+func (m *ManagerImpl) PrepareResources(pod *v1.Pod, container *v1.Container) (*ContainerInfo, error) {
+	if err := m.prepareContainerResources(pod, container); err != nil {
+		return nil, err
+	}
+
```

Comment:
nit: remove line

Reply:
This would trigger golangci wsl check failure:
```
pkg/kubelet/cm/dra/manager.go:192:2: return statements should not be cuddled if block has more than two lines (wsl)
        return m.getContainerInfo(pod, container)
```

Reply:
OK. I disagree with the linter in this case, but I don't feel too strongly so it's fine.

Reply:
let's leave it as it is now as I've fixed similar wsl checks in other places as well.